subsys/bluetooth/shell/iso.c

```diff
@@ -104,6 +106,35 @@ static int cmd_listen(const struct shell *shell, size_t argc, char *argv[])
 	return err;
 }
 
+static int cmd_peripheral(const struct shell *shell_ctx, size_t argc, char *argv[])
+{
+	struct bt_conn *conns[1];
+	struct bt_iso_chan *chans[1];
+
+	if (!default_conn) {
+		shell_error(shell_ctx, "Not connected");
+		return 0;
+	}
+
+	conns[0] = default_conn;
```

Comment:
I was wondering why we needed a check for `default_conn` for this one (it would make sense to me if the peripheral could setup this before any connection), but I guess we need to the set conns[0] pointer. 

Have you considered if it is possible to avoid doing this here?